MicroblogSpring/src/main/java/com/theironyard/MicroblogSpringController.java

```diff
@@ -0,0 +1,54 @@
+package com.theironyard;
+
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+
+import javax.servlet.http.HttpSession;
+import java.util.ArrayList;
+
+@Controller
+public class MicroblogSpringController {
+
+    static String loggedInUser = "loggedInUser";
+    static ArrayList<Message> messageList = new ArrayList<>();
+
+
+    @RequestMapping(path = "/", method = RequestMethod.GET)
+    public String home(Model model, HttpSession session) {
+        model.addAttribute("name", session.getAttribute(loggedInUser));
+        model.addAttribute("messageList", session.getAttribute("messageList"));
```

Comment:
No need to keep the entire message list in the session. It can become huge!  So, just use the local class static object (messageList) to add to the model.